packages/core/src/redux/slices/roomConnection/index.ts

```diff
@@ -309,3 +305,12 @@ startAppListening({
         }
     },
 });
+
+createReactor(
```

Comment:
I feel this should probably live on the localParticipant slice considering how much we are importing from there + that we are dispatching an action on that slice too. It still works here so not a blocker but perhaps a small tweak before merge 🤔

Reply:
yes it should! I moved it here before the refactoring and forgot to move it back. Will fix.

Reply:
done 👍